packages/react-native-renderer/src/ReactNativeFiberInspector.js

```diff
@@ -190,13 +190,21 @@ if (__DEV__) {
 
           closestInstance =
             internalInstanceHandle.stateNode.canonical._internalInstanceHandle;
+
+          // Note: this is deprecated and we want to remove it ASAP. Keeping it here for React DevTools compatibility for now.
+          const nativeViewTag = internalInstanceHandle.stateNode.canonical._nativeTag;
+
           nativeFabricUIManager.measure(
             internalInstanceHandle.stateNode.node,
             (x, y, width, height, pageX, pageY) => {
+              const inspectorData = getInspectorDataForInstance(
+                closestInstance,
+              );
               callback({
```

Comment:
Regarding why this might not work, is it possible that the code path a few lines up is actually the one being called?

https://github.com/facebook/react/blob/4a99c5c3a780d8178d9278c29d0b7f44be6a0dce/packages/react-native-renderer/src/ReactNativeFiberInspector.js#L183-L189

Reply:
I verified that the expected code-path is being hit, so that shouldn't be the root-cause. At that point it would require knowledge of/debugging the DevTools themselves (or the websocket bridget?) so I didn't get much further. But we are getting to this point, and we do have the correct nativeTag, so I'm not sure what's up